File: programers_Lv1/문자열 다루기 기본.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12918?language=python3
def solution(s):
    answer1 = s.isdecimal()
    answer2 = len(s) == 4 or len(s) == 6
    return answer1 and answer2

print(solution("1234"))

File: programers_Lv1/문자열 다루기 기본.js
// https://school.programmers.co.kr/learn/courses/30/lessons/12918#qna
function solution(s) {
  const numStr = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
  const numStrArr = [...numStr];
  let answer1 = numStrArr.some((el) => s.includes(el)) ? false : true;
  let answer2 = s.length === 4 || s.length === 6;
  return answer1 && answer2;
}

// console.log(solution("0x16"));
console.log(solution("123ABC"));
